seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSinkConfig.java

```diff
@@ -214,6 +214,13 @@ public class BaseSinkConfig {
                     .defaultValue(FileFormat.CSV)
                     .withDescription("File format type, e.g. csv, orc, parquet, text");
 
+    public static final Option<String> FILENAME_EXTENSION =
+            Options.key("filename_extension")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Override the default file name extensions with custom file name extensions. E.g. `.xml`, `.json`, `dat`, `.customtype`");
```

Comment:
Should we add some validataions to this param? Like user input `.`

Reply:
If this is what users want, I don't think we need to restrict it.